tests/test_html_parser.py

```diff
@@ -28,3 +28,13 @@ def test_annotated_text():
 
     assert to_unicode(document.paragraphs[1].sentences[0]) == "Tento text je tu aby vyplnil prázdne miesto v srdci súboru."
     assert to_unicode(document.paragraphs[1].sentences[1]) == "Aj súbory majú predsa city."
+
+def test_from_url():
+    url = "https://it.wikipedia.org/wiki/Parsing"
+    LANG = "italian"
+    parser = HtmlParser.from_url(url, Tokenizer(LANG))
```

Comment:
Maybe it would be better if you provide specific short text as an example instead of downloading the whole Wikipedia page. For example like this:

```py
text = """
<p>In <a href="/wiki/Informatica" title="Informatica">informatica</a>, il <b>parsing</b>, 
<b>analisi sintattica</b> o <b>parsificazione</b> è un processo che analizza un flusso 
continuo di dati in ingresso ...
"""
parser = HtmlParser.from_string(text, "https://it.wikipedia.org/wiki/Parsing", Tokenizer("italian"))
```